src/api/api.ts

```diff
@@ -0,0 +1,37 @@
+import { FastifyPluginAsync } from 'fastify';
+import { ValidationSchema } from '~/common/model-types/model-types.js';
+import {
+  Currency as CurrencyService,
+  Subscription as SubscriptionService,
+} from '../services/services.js';
+import { initCurrencyApi } from './currency/currency.api.js';
+import { initSubscriptionApi } from './subscription/subscription.api.js';
```

Comment:
Пропоную розділити власні й глобальні імпорти, як мені порекомендували і моїй роботі:
```suggestion
import { FastifyPluginAsync } from 'fastify';

import { ValidationSchema } from '~/common/model-types/model-types.js';
import {
  Currency as CurrencyService,
  Subscription as SubscriptionService,
} from '../services/services.js';
import { initCurrencyApi } from './currency/currency.api.js';
import { initSubscriptionApi } from './subscription/subscription.api.js';
```